echo_server.py

```diff
@@ -29,5 +33,39 @@ def start_server():
     finally:
         raise
 
+
+def response_ok(uri):
```

Comment:
The specification does not allow for a parameter to be passed in to this function.  It should just build a simple static 200 OK response.